PWGCF/JCorran/Tasks/flowJNUACreation.cxx

```diff
@@ -0,0 +1,150 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+// \brief   Task for the NUA correction with filtered data.
+// \author  Maxim Virta (maxim.virta@cern.ch)
+
+// Standard headers.
+#include <chrono>
+#include <string>
+#include <vector>
+#include <TRandom3.h>
+
+// O2 headers. //
+// The first two are mandatory.
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/ASoAHelpers.h"
+#include "CCDB/BasicCCDBManager.h"
+#include "Framework/HistogramRegistry.h"
+
+// O2 Physics headers. //
+#include "Common/DataModel/EventSelection.h"
+#include "Common/DataModel/FT0Corrected.h"
+#include "Common/DataModel/Multiplicity.h"
+#include "Common/DataModel/Centrality.h"
+#include "Common/Core/TrackSelection.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+
+#include "PWGCF/JCorran/Core/FlowJHistManager.h"
+
+// Namespaces and definitions.
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+using namespace std;
+
+using MyCollisions = soa::Join<aod::Collisions, aod::EvSels, aod::Mults,
+                               aod::FT0sCorrected, aod::CentFT0Ms,
+                               aod::CentFT0As, aod::CentFT0Cs, aod::CentFV0As,
+                               aod::CentFDDMs, aod::CentNTPVs>;
+
+using MyTracks = soa::Join<aod::Tracks, aod::TracksExtra, aod::TracksDCA>;
+
+struct flowJNUACreation {
```

Comment:
In here you are free to use `Task` as suffix if so you want
I did not complain about it before